client/src/components/About/index.jsx

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 
 const About = props => (
-  <div className="home game">
-    <div className="board">
-      <p>About placeholder</p>
+  <div className="main">
+    <div className="home game">
+      <div className="board">
```

Comment:
game and board classnames? This should probably just be main since it's not either of those things.